Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN /bin/bash -e /security_updates.sh && \
     && \
     add-apt-repository ppa:nginx/development -y && \
     apt-get update -yqq && \
-    apt-get install -yqq --no-install-recommends \
```

Comment:
why are we removing `--no-install-recommends` ?

Reply:
ah true, don't want to install unnecessary suggested packages?

Reply:
@ko-be @kangman With no recommends it wasn't installing the correct updated version of the lua packages. I guess I can figure out the dependency list and explicitly list them along w/ no recommends.

Reply:
```

Reply:
```